### ISSUE TITLE
Default selected date not working

### ISSUE THREAD

**Issue by Shekhar-Zealous** [jacobmischka/svelte-flatpickr#51]
Hello,

I am trying to set default selected date for the Datepicker but it is not working (Using Svelte). Anyone out there can you please help me. Thanks.
"Svelte Version": ^3.46.0
"svelte-flatpickr": "^3.2.6"

let datepickerOption = {
	minDate: new Date().fp_incr(1),
	time_24hr: true,
	allowInput: true,
	altInput: true,
	altFormat: 'F j, Y',
	dateFormat: 'Y-m-d',
        default: '2022-10-31'
};

<Flatpickr
	options={datepickerOption}
	on:change={(e) => handleBookingDateChange(e, i)}
	name="bookingDate"
	dateFormat="F j, Y"
	placeholder="Select Date"
	id="bookingDate"
/>

**Comment by torstent** [comment]
I think you should use the option `defaultDate` (according to https://flatpickr.js.org/options/)

But it is not working on my side with `defaultDate` when using `mode:range`

**stores.js**
```
import { writable } from 'svelte/store';

export const endDate = writable(new Date());
export const startDate = writable(new Date(new Date().setDate(new Date().getDate() - 1)));
```

**DateRange.svelte**
```
<script>
	import { startDate, endDate } from '../stores/stores.js';

	import Flatpickr from 'svelte-flatpickr';
	import 'flatpickr/dist/flatpickr.css';

	let value, formattedValue;

	const options = {
		enableTime: false,
	        mode: 'range',
        	//defaultDate: [$startDate.toLocaleString('de-DE', { dateStyle: 'short'}), $endDate.toLocaleString('de-DE', { dateStyle: 'short'})],
	        defaultDate: [$startDate, $endDate],
		onChange(selectedDates, dateStr) {
			//console.log('flatpickr hook', selectedDates, dateStr);
        	    	//$startDate = selectedDates[0];
            		//$endDate = selectedDates[1];
		}
	};

	$: console.log({ value, formattedValue });

	function handleChange(event) {
		const [ selectedDates, dateStr ] = event.detail;
		console.log({ selectedDates, dateStr });
	}

	function handleSubmit(event) {
		event.preventDefault();
		console.log(event.target.elements['date'].value);
	}
</script>

<h1>DateRange</h1>

<Flatpickr {options} bind:value bind:formattedValue on:change={handleChange} name="date" />

{$startDate.toLocaleString('de-DE', { dateStyle: 'short'})}
<br>
{$endDate.toLocaleString('de-DE', { dateStyle: 'short'})}
```

**Comment by jacobmischka** [comment]
Should be fixed now in `3.3.1`.
